src/components/course/data/constants.js

```diff
@@ -38,6 +38,7 @@ export const COURSE_MODES_MAP = {
   NO_ID_PROFESSIONAL: 'no-id-professional',
   AUDIT: 'audit',
   HONOR: 'honor',
+  PAID_EXECUTIVE_EDUCATION: 'paid-executive-education',
```

Comment:
[inform] FWIW, let's keep this as is for now, but we may want to revisit once we get some clarity if/when something comes of https://2u-internal.atlassian.net/browse/ENT-7165.